cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -3025,39 +3036,41 @@ def _verify_explicit_staff_lock_state(self, xblock_info, expected_state, path=No
     def test_empty_chapter(self):
         empty_chapter = self._create_child(self.course, 'chapter', "Empty Chapter")
         xblock_info = self._get_xblock_info(empty_chapter.location)
-        self._verify_visibility_state(xblock_info, VisibilityState.unscheduled)
+        self._verify_visibility_state(xblock_info, VisibilityState.live)
```

Comment:
why are these (and a few lines down) changes to the visibility state needed in this PR?

Reply:
I changed the start date for the generated course, so that the relative dates would be more reasonable. Until this PR, the start date was the default (2030), which translates to `unscheduled`.